src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
+/* eslint-disable max-len */
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum Color {
+  Red = 'red',
+  Blue = 'blue',
+  Green = 'green',
+}
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
 export class Triangle {
+  shape: Shape = Shape.Triangle;
+
+  color: string;
+
+  sides: [number, number, number];
+
+  constructor(color: string, side1: number, side2: number, side3: number) {
```

Comment:
<img width="227" alt="Screenshot 2023-10-29 at 15 53 01" src="https://github.com/mate-academy/ts_figures_classes/assets/135232001/38031111-02ab-4316-acce-a0c1a6ac7cb8">
Consider using this feature in your code